examples/ex11p.cpp

```diff
@@ -262,12 +262,13 @@ int main(int argc, char *argv[])
 #ifdef MFEM_USE_STRUMPACK
       if (sp_solver)
       {
-         STRUMPACKSolver * strumpack = new STRUMPACKSolver(argc, argv, MPI_COMM_WORLD);
+         STRUMPACKSolver * strumpack = new STRUMPACKSolver(MPI_COMM_WORLD, argc, argv);
```

Comment:
This seems like a breaking change. Maybe we can have both versions of the constructor, so we don't break backward compatibility.

Reply:
Fixed in [f5d0658](https://github.com/mfem/mfem/pull/3081/commits/f5d0658223984645535d01d5a68b18d653624fee).